docs/projects.md

```diff
@@ -167,7 +170,10 @@
 | [LitmusChaos](https://litmuschaos.io)   |  6/23/20        | 1/11/22           |[Incubating](https://github.com/cncf/toc/pull/569)|
 | [OpenMetrics](https://github.com/OpenObservability/OpenMetrics)|8/10/18| 2/2/2022 | [Incubating](https://github.com/cncf/toc/pull/615)|
 | [ChaosMesh](https://github.com/pingcap/chaos-mesh)|7/14/20| 2/16/2022 | [Incubating](https://github.com/cncf/toc/pull/683) |
-| [KubeVirt](https://kubevirt.io)         |  9/9/19        | 4/19/22           |[Incubating](https://github.com/cncf/toc/pull/715)|
```

Comment:
Did you mean to move Kubevirt into Graduating with this PR? :thinking:

Reply:
## Sandbox to Incubation Project Moves - line 149. The headers just didn't collapse properly in GH.